libgloss/arc/emsdp1.1.specs

```diff
@@ -0,0 +1,18 @@
+%rename link emsdp_link
+%rename link_gcc_c_sequence emsdp_link_gcc_c_sequence
+%rename startfile emsdp_startfile
+
+*link:
+%(emsdp_link) -T emsdp1.1.x%s -T arcv2elf-common.ld%s
```

Comment:
You don't need to have two linker scripts here, just use the `emsdp1.1.x`  and include the common one in this file.